pkgs/tools/networking/linux-router/default.nix

```diff
@@ -0,0 +1,89 @@
+# SOURCE: https://github.com/NixOS/nixpkgs/pull/92787/files
+
+{ stdenv, lib, fetchFromGitHub, makeWrapper
+
```

Comment:
```suggestion
```

Reply:
Would you still remove those two blank lines in the current version?

Reply:
Yes please (also line 11)

Reply:
Sorry, I did an `--amend` but forgot to stage before so there were actually no changes. :see_no_evil: Could you please have a look again? :sweat_smile: